sippy-ng/src/component_readiness/RegressedTestsModal.js

```diff
@@ -0,0 +1,245 @@
+import { Button, Grid, Tooltip, Typography } from '@mui/material'
+import { CompReadyVarsContext } from './CompReadyVars'
+import { DataGrid, GridToolbar } from '@mui/x-data-grid'
+import { formColumnName, sortQueryParams } from './CompReadyUtils'
+import { Link } from 'react-router-dom'
+import { safeEncodeURIComponent } from '../helpers'
+import CompSeverityIcon from './CompSeverityIcon'
+import Dialog from '@mui/material/Dialog'
+import PropTypes from 'prop-types'
+import React, { Fragment, useContext } from 'react'
+
+// Construct a URL with all existing filters plus testId, environment, and testName.
+// This is the url used when you click inside a TableCell on page4 on the right.
+// We pass these arguments to the component that generates the test details report.
+function generateTestReport(
+  testId,
+  platform,
+  upgrade,
+  arch,
+  network,
+  variant,
+  filterVals,
+  componentName,
+  capabilityName,
+  testName
+) {
+  const environment = {
+    network: network,
+    upgrade: upgrade,
+    arch: arch,
+    platform: platform,
+    variant: variant,
+  }
+  const environmentVal = formColumnName(environment)
+  const { expandEnvironment } = useContext(CompReadyVarsContext)
+  const safeComponentName = safeEncodeURIComponent(componentName)
+  const safeTestId = safeEncodeURIComponent(testId)
+  const safeTestName = safeEncodeURIComponent(testName)
+  const safePlatform = safeEncodeURIComponent(platform)
+  const safeUpgrade = safeEncodeURIComponent(upgrade)
+  const safeArch = safeEncodeURIComponent(arch)
+  const safeNetwork = safeEncodeURIComponent(network)
+  const safeVariant = safeEncodeURIComponent(variant)
+  const retUrl =
+    '/component_readiness/test_details' +
+    filterVals +
+    `&testId=${safeTestId}` +
+    expandEnvironment(environmentVal) +
+    `&component=${safeComponentName}` +
+    `&capability=${capabilityName}` +
+    `&platform=${safePlatform}` +
+    `&upgrade=${safeUpgrade}` +
+    `&arch=${safeArch}` +
+    `&network=${safeNetwork}` +
+    `&variant=${safeVariant}` +
+    `&testName=${safeTestName}`
+
+  return sortQueryParams(retUrl)
+}
+
+export default function RegressedTestsModal(props) {
+  // define table columns
+  const columns = [
+    {
+      field: 'component',
+      headerName: 'Component',
+      flex: 20,
+      renderCell: (param) => (
+        <div className="test-name">
+          <Tooltip title={'Component'}>{param.value}</Tooltip>
```

Comment:
Sorry I meant to have the tooltip show the value.  In the screenshot I shared before all the values were truncated with ellipsis.  Like this:

```
<Tooltip title={param.value}>{param.value}</Tooltip>
```

But now you also wrapped these in a `<div className="test-name">` which turns off the truncate.  You can probably drop the tooltip now, depending which view you prefer.